pytest/bitops_test.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+
+import arkouda as ak
+import pytest
+
+
+@pytest.fixture(scope="function", autouse=True)
+def bitops_data():
+    pytest.bitops_data = {
+        'A': ak.arange(10),
+        'B': ak.cast(ak.arange(10), ak.uint64),
+        'EDGE_CASES': ak.array([-(2**63), -1, 2**63 - 1]),
+        'EDGE_CASES_UINT': ak.cast(ak.array([-(2**63), -1, 2**63 - 1]), ak.uint64)
+    }

```

Comment:
The specific scope declaration was the result of the fixture output of `pdarray` deletion, not the var/return decision. Since the scope is `function`, recreation will be the case regardless of using `return` or this method. I thought this way was cleaner which is the only reason I did it this way. I'm not opposed to changing it if that's the path we want to take